packages/editor-ui/src/components/RunData.vue

```diff
@@ -1253,47 +1255,17 @@ export default defineComponent({
 			const itemsLabel = itemsCount > 0 ? ` (${itemsCount} ${items})` : '';
 			return option + this.$locale.baseText('ndv.output.of') + (this.maxRunIndex + 1) + itemsLabel;
 		},
-		getDataCount(
-			runIndex: number,
-			outputIndex: number,
-			connectionType: ConnectionTypes = NodeConnectionType.Main,
-		) {
-			if (this.pinData) {
-				return this.pinData.length;
-			}
-
+		getDataCount(runIndex: number) {
 			if (this.node === null) {
 				return 0;
 			}
 
 			const runData: IRunData | null = this.workflowRunData;
```

Comment:
Up to you - Untouched but since we're here...

```suggestion
			const runData = this.workflowRunData;
```